runner_manager/logging.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from runner_manager.models.settings import Settings
+
+settings = Settings()
```

Comment:
Otherwise you are reading the configs all over again each time you call it
```suggestion
from runner_manager.dependencies import get_settings

settings = get_settings()
```